azkaban-exec-server/src/main/java/azkaban/execapp/GuiceConstants.java

```diff
@@ -0,0 +1,8 @@
+package azkaban.execapp;
+
+public class GuiceConstants {
+
+  public static final String EXEC_JETTY_SERVER = "ExecServer";
```

Comment:
The reason is that otherwise one might easily create a circular dependency between a module class and the classes to instantiate (that use `@Named` in constructor args). Do you want me to move these any way?